.ci/scripts/button-left-click.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/python3
+
+import argparse
+import uinput
+import time
+
+DEVICE_READY_TIMEOUT = 3
+DISPLAY_WIDTH = 1920
+DISPLAY_HEIGHT = 1080
+
+
+"""
+Simulate button left click at (start_x, start_y).
+
+When offset_x or offset_y are set, their value is added at the cursor position
+before release the button.
+"""
+def button_left_click(device, start_x, start_y, offset_x=0, offset_y=0, delay=1):
+    if (delay > 0):
```

Comment:
does makes sense this delay? What is its usefulness?. It could be more interesting a delay between the touch down and the touch up.

Reply:
I added this delay because the test changes the background color to green if the 'click' event is correct, and then changes the color back to white (1 second delay by default). So adding a 1 second delay between tests it will make possible to run several tests without reloading the page. In the end, I'm reloading the page for each of the tests, which I think it makese sense.

In conclusion, I will remove this delay :)